src/main/java/frc/robot/commandgroups/Outtake.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commandgroups;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.robot.Constants;
+import frc.robot.subsystems.conveyor.Conveyor;
+import frc.robot.subsystems.conveyor.commands.Convey;
+import frc.robot.subsystems.conveyor.commands.FlapCommand;
+import frc.robot.subsystems.intake.Intake;
+import frc.robot.subsystems.intake.commands.IntakeCargo;
+import frc.robot.subsystems.shooter.Shooter;
+import frc.robot.subsystems.shooter.commands.Shoot;
+
+import java.util.OptionalDouble;
+import java.util.function.BooleanSupplier;
+
+public class Outtake extends ParallelCommandGroup {
+
+    public Outtake(Intake intake,
+                   Conveyor conveyor,
+                   Shooter shooter,
+                   BooleanSupplier condition) {
+        addCommands(
+                new FlapCommand(conveyor, Conveyor.FlapMode.Open),
+                new Convey(conveyor, () -> Constants.Conveyor.DEFAULT_POWER * (condition.getAsBoolean() ? 1 : -1)),
```

Comment:
There is a problem, both `FlapCommand` and `Convey` require the conveyor

Reply:
I have to separate the flap from the conveyor